src/main/steps/sequence.test.ts

```diff
@@ -30,7 +35,7 @@ describe('Sequence', () => {
 
     it('returns current step if there is no next step', () => {
       mockReq.originalUrl = HAS_RELATIONSHIP_BROKEN_URL;
-      expect(getNextStepUrl(mockReq)).toBe(HAS_RELATIONSHIP_BROKEN_URL);
+      expect(getNextStepUrl(mockReq)).toBe(RELATIONSHIP_DATE_URL);
```

Comment:
I think this test should be:

```ts
      const lastUrl = RELATIONSHIP_DATE_URL;
      mockReq.originalUrl = lastUrl;
      expect(getNextStepUrl(mockReq)).toBe(lastUrl);
```

Reply:
yes but i think original should be the url before the last url. I assume youre going for that idea
`const lastUrl = RELATIONSHIP_DATE_URL;
      mockReq.originalUrl = HAS_RELATIONSHIP_BROKEN_URL;
      expect(getNextStepUrl(mockReq)).toBe(lastUrl);`